tests/components/recollect_waste/test_config_flow.py

```diff
@@ -45,6 +46,29 @@ async def test_invalid_place_or_service_id(hass):
         assert result["errors"] == {"base": "invalid_place_or_service_id"}
 
 
+async def test_options_flow(hass):
+    """Test config flow options."""
+    conf = {CONF_PLACE_ID: "12345", CONF_SERVICE_ID: "12345"}
+
+    config_entry = MockConfigEntry(domain=DOMAIN, unique_id="12345, 12345", data=conf)
+    config_entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.recollect_waste.async_setup_entry", return_value=True
+    ):
+        result = await hass.config_entries.options.async_init(config_entry.entry_id)
```

Comment:
We need to set up the config entry before starting the options flow. Use `hass.config_entries.async_setup`.

Reply:
@MartinHjelmare Can you clarify? I've already set up a mock config entry above this.

Reply:
The mock config entry has just been added to home assistant. It's not setup.

When we set up the config entry we make sure to load the config flow module. The test would fail if run in isolation otherwise when we try to start the options flow.

Reply:
Got it. Let me know if caaffaf does what you want?